dynamiq/nodes/node.py

```diff
@@ -329,6 +336,40 @@ def validate_depends(self, depends_result):
                     depend=dep, depends_result=depends_result
                 )
 
+    @property
+    def input_schema_params(self) -> list[tuple]:
+        """
+        Return list of input parameters along with their type and description.
+        """
+        params = []
+        for name, field in self.input_schema.model_fields.items():
```

Comment:
Why just `self.input_schema.model_fields` dict not fit as the return of this func. It handles all other Field attrs (e.g title, frozen, required, ....)?